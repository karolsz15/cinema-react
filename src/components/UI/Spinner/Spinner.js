import React from 'react';
import classes from './Spinner.module.css';

const Spinner = () => (
    <div className={classes.Lds}>
      <div />
      <div />
    </div>
  );

export default Spinner;
