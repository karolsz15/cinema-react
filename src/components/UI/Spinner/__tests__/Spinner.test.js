import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../Spinner';

let wrapper = shallow(<Spinner />);

describe('Spinner component', () => {
  it('should render the Spinner correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});