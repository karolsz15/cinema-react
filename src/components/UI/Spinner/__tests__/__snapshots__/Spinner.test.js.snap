// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should render the Spinner correctly 1`] = `
<div
  className="Lds"
>
  <div />
  <div />
</div>
`;
